<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInterface</class>
 <widget class="QDialog" name="MainInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1350</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1350</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>900</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1350</width>
     <height>900</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1350</width>
     <height>900</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>402</width>
     <height>320</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="ButtonPlay">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>75</height>
       </size>
      </property>
      <property name="text">
       <string>游玩</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonEdit">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>75</height>
       </size>
      </property>
      <property name="text">
       <string>编辑器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonBoard">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>75</height>
       </size>
      </property>
      <property name="text">
       <string>排行榜</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonQuit">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>75</height>
       </size>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
